pymc3/smc/smc.py

```diff
@@ -418,6 +419,85 @@ def sample_settings(self):
         return stats
 
 
+class MH(SMC_KERNEL):
+    """Metropolis-Hastings SMC kernel"""
+
+    def __init__(self, *args, n_steps=25, **kwargs):
+        """
+        Parameters
+        ----------
+        n_steps: int
+            The number of steps of each Markov Chain.
+        """
+        super().__init__(*args, **kwargs)
+        self.n_steps = n_steps
+
+        self.proposal_dist = None
+        self.proposal_scales = np.ones((self.draws, 1))
+        self.acc_rate = None
+        self.first_iteration = True
+
+    def setup_kernel(self):
+        """Proposal dist is just a Multivariate Normal with unit identity covariance.
+        Dimension specific scaling is provided by self.proposal_scales and set in self.tune()
+        """
+        posterior_ndim = self.population.shape[1]
+        self.proposal_dist = multivariate_normal(
+            mean=np.zeros(posterior_ndim),
+            cov=np.eye(posterior_ndim),
+        )
+
+    def tune(self):
+        """Update proposal scales for each particle dimension"""
+        if not self.first_iteration:
+            for i, (scale, acc_rate) in enumerate(zip(self.proposal_scales, self.acc_rate)):
+                self.proposal_scales[i] = metropolis_tune(scale, acc_rate)
```

Comment:
Not sure how relevant, but in the old days òf SMC with MH kernel we used to have

```python
ave_scaling = np.exp(np.log(self.scalings.mean()) + (self.acc_per_chain.mean() - 0.234))
self.scalings = 0.5 * (
    ave_scaling + np.exp(np.log(self.scalings) + (self.acc_per_chain - 0.234))
) 

```
With the scaling initialized at.
```python
self.scalings = np.ones(self.draws) * min(1, 2.38 ** 2 / dimension)
```

Reply:
Reindex `scalings` and `acc_per_chain`, and mention this in general Kernel...